src/groundStation/system.py

```diff
@@ -76,6 +76,61 @@ def __init__(self):
             'LAST': 31
         }
         self.SERVICES = {
+            'SCHEDULER': {
+                'port': 15,  # share a port with EPS time service
```

Comment:
I think sharing ports is okay, but you shouldn't mention it. Has a chance of confusing people

Reply:
pushed